### PR TITLE
Include string header to resolve undefined type error

### PR DESCRIPTION
**Pull request by zoq**
I guess you only run into the issue if you don't use cotire.

### CODE REVIEW

**Review by rcurtin — APPROVED** [review]
I'm not sure what happened, but I thought I already approved this once?  But it looks like the approval never actually posted.  Anyway, nice fix!

**Review by Yashwants19 — APPROVED** [review]
Thanks for the patch.
This looks good to me. 👍

**Pull request merged**
